test/write.spec.ts

```diff
@@ -137,7 +140,7 @@ jest.mock('../src/git', () => ({
     },
 }));
 
-describe('writeBenchmark()', function () {
+describe('writeBenchmark() github.com', function () {
```

Comment:
If I'm not mistaken this whole test suite code is duplicated for the GHE case. We can use [describe.each](https://jestjs.io/docs/api#describeeachtablename-fn-timeout) to avoid this duplication
```suggestion
describe.each([
  { serverUrl: 'https://github.com' },
  { serverUrl: 'https://github.enterprise.corp' }
])('writeBenchmark() - $serverUrl', function ({ serverUrl }) {
```

Reply:
Adapted [test](https://github.com/mjochum/github-action-benchmark/blob/fix/serverUrls/test/write.spec.ts#L143) accordingly 👍